zone/attack.cpp

```diff
@@ -2186,6 +2186,19 @@ bool Client::Death(Mob* killer_mob, int64 damage, uint16 spell, EQ::skills::Skil
 	return true;
 }
 
+bool Client::CheckDeath()
```

Comment:
This has a bit of a confusing name for what it's actually doing.  Maybe something like CheckIfAlreadyDead instead of CheckDeath.

Reply:
CheckDeath kind of implies you're checking if the client is dead instead of checking if they have triggered the dying logic before.